.gitignore

```diff
@@ -100,6 +100,9 @@ doc/user/README.md
 # Ignore .rb files at the root, it's temporary files to debug some issue
 /*.rb
 
+# Ignore Seafoam generated illustrations
+graph.pdf
```

Comment:
How about `/graph*.pdf`?
That would catch things like `graph-after.pdf` and `graph_mc.pdf` which I have locally.

The `/` is important to only match at the root and not in subdirs.

Reply:
First time I'm seeing those names! How are they generated. Also- how about `/*.pdf` or `*.pdf`, maybe?

Reply:
> How are they generated?

Manual copying with `cp` to compare multiple graphs :)
`/*.pdf` sounds fine, yes.